### PR TITLE
Enforce black

### PR DESCRIPTION
**Pull request by vtemian**
Pretty straight forward, [black](https://github.com/psf/black) is enforced in order to simplify all the debates regarding code formating.

Integrated from https://github.com/code4romania/seismic-risc/commit/0a7de6cd483df4b5eb8b3bdd57be33f1271701ec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,16 +45,23 @@ Catalog Politic - Declaratii de avere - aims to automate the process of parsing
...
-Python 3.5+    
+Python 3.5+
 Please follow [the Python style guide](python_style_guide.md).
 
+In order to solve the formatting debates, [black](https://github.com/psf/black) is enforced.
+Before submiting a PR, you can run
+```bash
+black --line-length 80 --target-version py37 .
+```
+and you're code will be automatically formatted.
```
Comment (catileptic) [inline README.md]: _your_ code

File: README.md [hunk 2]
```diff
@@ -69,14 +76,14 @@ Political Catalogue - Asset Declaration is a web application.
 
 ### Database technology & provider
 
-This remains currently undecided.   
+This remains currently undecided.
```
Comment (catileptic) [inline README.md]: Could you please add [PostgreSQL](https://www.postgresql.org), here?

File: README.md [hunk 3]
```diff
@@ -69,14 +76,14 @@ Political Catalogue - Asset Declaration is a web application.
...
 
 ## Repos and projects
 
-[Moonsheep on GitHub](https://github.com/themoonsheep)    
+[Moonsheep on GitHub](https://github.com/themoonsheep)
 [PyBossa on GitHub](https://github.com/Scifabric/pybossa)
```
Comment (catileptic) [inline README.md]: Can you remove this?

**Comment by vtemian** [comment]
@catileptic done.

**Pull request merged**